pl_examples/basic_examples/conf/pl/logger/tensorboard.yaml

```diff
@@ -0,0 +1,5 @@
+# @package _group_
+cls: pytorch_lightning.loggers.tensorboard.TensorBoardLogger
+params:
+  name: ${hydra:job.name}
+  save_dir: /tmp/
```

Comment:
I have not actually used Tensorboard.
does it make sense for the tb logs to exist the job working directory for each run or is it more convenient to put all the logs in the same directory like seems to be doing here?

Reply:
Usually it make sense to put them in the same directory as they can then be compared across experiments. Tensorboard, see `https://github.com/tensorflow/tensorboard#usage`

Reply:
Also I want to think of a more descriptive name here.

Reply:
They cannot be easily compared if they are in the same top level directory```
multirun/
  sweep1/
    job1/
    job2/
   ...
```

Reply:
I think that maybe possible, I will check...I am not much of a tensorboard user

Reply:
This does work, so we can just place the logger files in the output directory.